kobo/device.go

```diff
@@ -343,7 +352,7 @@ func (d Device) CoverSize(t CoverType) image.Point {
 	}
 
 	switch d.CodeNames().Family() {
-	case CodeNameDragon, CodeNameSnow:
+	case CodeNameDragon, CodeNameSnow, CodeNameEuropa:
 		return image.Pt(1080, 1440)
```

Comment:
@NiLuJe, can you check this for N3_FULL covers? I was expecting it to match the device resolution, but based on skimming the code, it shares the size with Dragon instead...

Reply:
*sigh*. Yup.

```
/mnt/onboard/.kobo-images/39/72/file____mnt_onboard_kfmon_png - N3_FULL.parsed JPEG 1080x1440 1080x1440+0+0 8-bit sRGB 99542B 0.000u 0:00.000
```

Screen is actually 1404x1872, FWIW.

(IIRC, the last time this happened, I think I had managed to convince @davidfor to do the right thing in the Calibre driver instead of following nickel ;)).

Reply:
And calibre does indeed do the right thing:

https://github.com/kovidgoyal/calibre/blob/01d18e5ddaa79f5a8fd06ee10d0cab6022d35a5a/src/calibre/devices/kobo/driver.py#L3434-L3435

\o/

Reply:
@gtalusan might want to fix this...

@NiLuJe, I guess I'll probably change it to use the correct resolution here, since the point of koboutils (and covergen, which uses it) is to do the best thing rather than exactly match libnickel. IIRC, this is what we did last time too.

BTW, I also noticed some other changes in the cover sizing in libnickel, so I'm going to need to figure out when those were introduced, and possibly add them to here.

@NiLuJe, to make things a bit quicker and so I can ensure I read it correctly, could you send a list of the cover types and sizes you have from the Elipsa?

Reply:
The way the source is sized, this *should* match nickel's max values:

```
/mnt/onboard/.kobo-images/39/72/file____mnt_onboard_kfmon_png - N3_FULL.parsed JPEG 1080x1440 1080x1440+0+0 8-bit sRGB 99542B 0.000u 0:00.000
/mnt/onboard/.kobo-images/39/72/file____mnt_onboard_kfmon_png - N3_LIBRARY_FULL.parsed JPEG 355x473 355x473+0+0 8-bit sRGB 17061B 0.000u 0:00.000
/mnt/onboard/.kobo-images/39/72/file____mnt_onboard_kfmon_png - N3_LIBRARY_GRID.parsed JPEG 149x198 149x198+0+0 8-bit sRGB 4940B 0.000u 0:00.000
```